test/points_test.cpp

```diff
@@ -15,14 +15,7 @@ TEST_CASE("Points tests", "[Point]")
         auto points = Points(3, 4);
         REQUIRE(points.PointFormatID() == 3);
         REQUIRE(points.PointRecordLength() == 38);
-        REQUIRE(points.GetPoints().empty());
-
-        points = Points(3, 4, 10);
-        REQUIRE(points.PointFormatID() == 3);
-        REQUIRE(points.PointRecordLength() == 38);
-        REQUIRE(points.GetPoints().size() == 10);
-        for (const auto &point : points.GetPoints())
-            REQUIRE(point.PointFormatID() == 3);
```

Comment:
We don't need that constructor?

Reply:
I removed the constructor and added an explicit Reserve() function for that purpose. The constructor was giivng me headaches